configs/piksiv3_defconfig

```diff
@@ -61,11 +61,13 @@ BR2_PACKAGE_OPENSSH=y
 BR2_PACKAGE_PICOCOM=y
 BR2_PACKAGE_PPPD=y
 BR2_PACKAGE_SOCAT=y
+BR2_PACKAGE_RSYNC=y
 BR2_PACKAGE_SUDO=y
 BR2_PACKAGE_KMOD=y
 BR2_PACKAGE_KMOD_TOOLS=y
 BR2_PACKAGE_MONIT=y
 BR2_TARGET_ROOTFS_INITRAMFS=y
+BR2_PACKAGE_DOSFSTOOLS_FSCK_FAT=y
```

Comment:
Ah yes, I was going to add an fsck in because in practice the filesystem is in rough shape and the logs seem incomplete.  Perhaps with the plan of doing journaling file systems this is unnecessary.  I will remove.